Makefile

```diff
@@ -23,7 +23,7 @@ all:
 
 install: dependencies stack copyfiles plugin-dependencies plugins version
 
-copyfiles: addman
```

Comment:
This told make to run `addman` before `copyfiles`. `help2man` calls dokku and thus needs to be in the `$PATH` first